worker/provisioner/provisioner_task.go

```diff
@@ -725,9 +713,10 @@ func (task *provisionerTask) maintainMachines(machines []*apiprovisioner.Machine
 
 // AvailabilityZoneMachine exposed for testing
```

Comment:
Comment should explain what this struct is used for - aids in readability etc

Reply:
comment added.